iac-common/src/main/java/org/sonar/iac/common/yaml/tree/FileTree.java

```diff
@@ -20,7 +20,11 @@
 package org.sonar.iac.common.yaml.tree;
 
 import java.util.List;
+import javax.annotation.CheckForNull;
 
 public interface FileTree extends YamlTree {
   List<YamlTree> documents();
+
+  @CheckForNull
+  String template();
```

Comment:
From quick view on it, it looks like only `"helm`` and `null` is used here. Maybe an enum will be better?